content-api-post.js

```diff
@@ -27,10 +27,18 @@ exports.handler = function (event, context, callback) {
   const data = JSON.parse(event.body);
   const decodedImage = Buffer.from(data.body, 'base64');
 
-  if (!decodedImage || !data.accountId || !data.type) {
+  if (!decodedImage || (!data.accountId && !data.championFundId) || !data.type) {
     return fail('Missing required metadata', 400, callback);
   }
 
+  if (!data.accountId && data.championFundId && data.ccampaignId) {
```

Comment:
@tbg-jasonfung what's the rationale for having all 3 parts here? Given the below checks we're not combining account + fund, shouldn't this just check fo campaign + fund?

Reply:
@NoelLH Nicee, I've also removed the duplicate return fail message and bunged these checks all into 1 line